Feliz/Styles.fs

```diff
@@ -4314,6 +4314,20 @@ module style =
         /// and to its initial value if not.
         static member inline unset = Interop.mkStyle "outlineStyle" "unset"
 
+    /// The object-fit CSS property sets how the content of a replaced element, such as an <img> or <video>, should be resized to fit its container.
+    [<Erase>]
+    type objectFit =
+        /// The replaced content is scaled to maintain its aspect ratio while fitting within the element's content box. The entire object is made to fill the box, while preserving its aspect ratio, so the object will be "letterboxed" if its aspect ratio does not match the aspect ratio of the box.
+        static member inline contain = Interop.mkStyle "object-fit" "contain"
+        /// The replaced content is sized to maintain its aspect ratio while filling the element's entire content box. If the object's aspect ratio does not match the aspect ratio of its box, then the object will be clipped to fit.
+        static member inline cover = Interop.mkStyle "object-fit" "cover"
+        /// The replaced content is sized to fill the element's content box. The entire object will completely fill the box. If the object's aspect ratio does not match the aspect ratio of its box, then the object will be stretched to fit.
+        static member inline fill = Interop.mkStyle "object-fit" "fill"
+        /// The replaced content is not resized.
+        static member inline none = Interop.mkStyle "object-fit" "none"
+        /// The content is sized as if none or contain were specified, whichever would result in a smaller concrete object size.
+        static member inline scaleDown = Interop.mkStyle "object-fit" "scale-down"
```

Comment:
I think that the key needs to be camelCase instead of hyphenated
```suggestion
        static member inline scaleDown = Interop.mkStyle "objectFit" "scale-down"
```